### PR TITLE
SFR-937 Upgrade DS to 0.4.1 

### PR DESCRIPTION
**Pull request by hokei**
[Jira Ticket](http://jira.nypl.org/browse/SFR-XXX)

### This PR does the following:
- upgrades SFR to use Design System 0.4.0
- New overrides to beautify

### Testing requirements & instructions: 
-  Check for regressions in front end app

### CODE REVIEW

File: src/app/components/About/About.jsx [hunk 1]
```diff
@@ -71,21 +71,21 @@ All the materials in Digital Research Books Beta are completely free to read and
                     The material in Digital Research Books Beta are drawn from several
                                         public sources, mainly
               {' '}
-              <DS.UnderlineLink url="https://www.hathitrust.org/">HathiTrust</DS.UnderlineLink>
+              <DS.Link href="https://www.hathitrust.org/">HathiTrust</DS.Link>
```
Comment (mwbenowitz) [inline src/app/components/About/About.jsx:74]: Is there a difference between the `href` and `url` attributes used on `DS.Link`. I see both and assume they do the same thing but maybe should be aligned for consistency?

Comment (hokei) [reply src/app/components/About/About.jsx:74]: url is never used now, it's always href.

Comment (mwbenowitz) [reply src/app/components/About/About.jsx:74]: Look down in the `License` file looks like its showing up there

Comment (hokei) [reply src/app/components/About/About.jsx:74]: omg the license page wasn't working at all, thank you for catching this.

**Pull request merged**